x-pack/plugins/observability_solution/entity_manager/server/lib/entities/install_entity_definition.test.ts

```diff
@@ -52,15 +52,15 @@ const assertHasCreatedDefinition = (
     id: generateHistoryIngestPipelineId(builtInServicesFromLogsEntityDefinition),
     processors: expect.anything(),
     _meta: {
-      definitionVersion: '0.1.0',
+      definitionVersion: '1.0.0',
```

Comment:
this is addressed here https://github.com/elastic/kibana/pull/188351/files#r1679536036